cmd/piper.go

```diff
@@ -49,10 +50,11 @@ type GeneralConfigOptions struct {
 	GCSSubFolder         string
 }
 
-// HookConfiguration contains the configuration for supported hooks, so far Sentry and Splunk are supported.
+// HookConfiguration contains the configuration for supported hooks, so far ANS, Sentry and Splunk are supported.
 type HookConfiguration struct {
 	SentryConfig SentryConfiguration `json:"sentry,omitempty"`
 	SplunkConfig SplunkConfiguration `json:"splunk,omitempty"`
+	ANSConfig    ans.Configuration   `json:"ans,omitempty"`
```

Comment:
That yaml is correct. Although we don't want to have secrets in the yaml. So instead it is encouraged to use env variables. If someone sets it here, however, it will be used; rather than the env var.
Does that answer your question?